src/EasyCaching.Redis/DefaultRedisCachingProvider.Async.cs

```diff
@@ -53,7 +53,8 @@ public override async Task<object> BaseGetAsync(string cacheKey, Type type)
         /// <param name="dataRetriever">Data retriever.</param>
         /// <param name="expiration">Expiration.</param>
         /// <typeparam name="T">The 1st type parameter.</typeparam>
-        public override async Task<CacheValue<T>> BaseGetAsync<T>(string cacheKey, Func<Task<T>> dataRetriever, TimeSpan expiration)
+        public override async Task<CacheValue<T>> BaseGetAsync<T>(string cacheKey, Func<Task<T>> dataRetriever,
+            TimeSpan expiration)
```

Comment:
Maybe not change uncorrelated code for the issue.

Reply:
auto-formatted, will revert